packages/blocks/src/lib/navigation-map/index.svelte

```diff
@@ -36,10 +36,10 @@ export let waypoints: Waypoint[] = [];
 export let obstacles: Obstacle[] = [];
 
 /** The initial tab to show */
-export let tab: NavigationTabType = NavigationTab.Waypoints;
+export let tab: string = NavigationTab.Waypoints;
 
 /** The visible set of tabs */
-export let tabs = [NavigationTab.Waypoints, NavigationTab.Obstacles];
+export let tabs: string[] = [NavigationTab.Waypoints, NavigationTab.Obstacles];
```

Comment:
What was the reasoning for loosening this to `string` rather than adding `Attributes` to the union?